backend/routes/hello.php

```diff
@@ -0,0 +1,10 @@
+<?php
+require('../includes/header.php');
+
+$status_code = 200;
+$response = [
+    'message' => 'Hello world'
+];
+
+require('../includes/return.php');
```

Comment:
To jest mało przejrzyste i może być problematyczne jeżeli `return.php` stanie się bardziej rozbudowany.

Pierwszy raz coś takiego widzę. Zazwyczaj spotykałem się że jest funkcja np. `return($status_code, $response)`. Nie znam się aż tak na PHP, więc jeżeli tak się robi to możemy to zostawić - zostawiam tą decyzję Tobie.